pkgs/tools/admin/novnc/default.nix

```diff
@@ -0,0 +1,26 @@
+{ lib, stdenv, fetchFromGitHub }:
+
+stdenv.mkDerivation rec {
+  pname = "novnc";
+  version = "1.3.0";
+
+  src = fetchFromGitHub {
+    owner = "novnc";
+    repo = "noVNC";
+    rev = "v${version}";
+    sha256 = "sha256-Z+bks7kcwj+Z3uf/t0u25DnGOM60QhSH6uuoIi59jqU=";
+  };
+
+  installPhase = ''
+    mkdir -p $out
+    cp -r $src/* $out
+  '';
+
+  meta = with lib; {
+    description = "VNC client web application";
+    homepage = "https://novnc.com";
+    license = with licenses; [ mpl20 ofl bsd3 bsd2 mit ];
+    maintainers = with maintainers; [ neverbehave ];
+    platforms = [ ];
```

Comment:
This is a server program right? but leaving meta.platforms `[]` implies none of platforms are guarantee/intended to work. This doesn't make any sense. This should be at least `[ "x86_64-linux" ]` as you tested on it. Otherwise this package should be under data to mean the package is not runnable.

Reply:
this is meant to serve using web server like nginx. So I guess it should be under data section. Which part exactly should I place it?

Reply:
I placed in data/misc. Not sure if it is the best suited place tho